src/message/MessageContainer.js

```diff
@@ -35,7 +30,7 @@ export default class MessageContainer extends React.PureComponent {
         twentyFourHourTime={twentyFourHourTime}
         selfEmail={auth.email}
         doNarrow={doNarrow}
-        onPress={this.handlePress}
+        onLongPress={() => handleLongPress(message)}
```

Comment:
Extract that to a handleLongPress function in the class, and then onLongPress={this.handleLongPress}
`() =>` creates a new function for each instance.